README.md

```diff
@@ -41,12 +41,7 @@ Then, run the Rails web server:
 
 ### To deploy to App Engine Managed VMs
 
-    $ gcloud preview app deploy app.yaml --set-default
-
-This application uses a custom `Dockerfile` based on the [`google/ruby`][] image.
-
-The `Dockerfile` also runs `bundle install --without development:test` so that only
-production gems will be installed in production.
+    $ gcloud preview app deploy app.yaml --promote
 
 [google/ruby]: https://registry.hub.docker.com/u/google/ruby-runtime/
```

Comment:
Oh, yep, actually I was removing references to this link, but need to remove the actual link.